sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/compiler/CxxCompilerSensor.java

```diff
@@ -47,6 +47,7 @@
   public static final String REPORT_CHARSET_DEF = "sonar.cxx.compiler.charset";
   public static final String PARSER_KEY_DEF = "sonar.cxx.compiler.parser";
   public static final String DEFAULT_PARSER_DEF = CxxCompilerVcParser.KEY;
+  public static final String DEFAULT_CHARSET_DEF = "UTF-16";
```

Comment:
UTF-16 is required for the VS2010 html reports. VS2010 was the first version for VC++ reports support and the compiler command syntax did not change.
Using UTF-8 for the buildlog Parser is OK but we should be consistent and also update https://github.com/wenns/sonar-cxx/blob/master/sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/compiler/CxxCompilerVcParser.java